### PR TITLE
Devel

### PR DESCRIPTION
**Pull request by hpsaturn**

### CODE REVIEW

File: src/main.cpp [hunk 1]
```diff
@@ -36,6 +36,8 @@
 
 #ifdef WEMOSOLED // display via i2c for WeMOS OLED board
 U8G2_SSD1306_128X64_NONAME_F_SW_I2C u8g2(U8G2_R0, 4, 5, U8X8_PIN_NONE);
+//#elif ESP32Sboard // display via i2c for ESP32S board
```
Comment (hpsaturn) [inline src/main.cpp]: Maybe @danielbernalb these lines are a unnecessary comments?

File: src/main.cpp [hunk 2]
```diff
@@ -142,25 +148,25 @@ char getLoaderChar(){
...
  **/
 void sensorLoop(){
+  Serial.print("-->[HPMA] read..");
   int try_sensor_read = 0;
   String txtMsg = "";
   while (txtMsg.length() < 32 && try_sensor_read++ < SENSOR_RETRY) {
     while (hpmaSerial.available() > 0) {
       char inChar = hpmaSerial.read();
       txtMsg += inChar;
-      Serial.print("-->[HPMA] read "+String(getLoaderChar())+"\r");
```
Comment (hpsaturn) [inline src/main.cpp]: @danielbernalb these a lines was approved in a previous PR, we need discuss it. Please ever review the PRs for not will do unnecessary reverts in the future.

File: src/main.cpp [hunk 3]
```diff
@@ -243,9 +255,9 @@ void humidityLoop() {
 void batteryloop() {
 #ifdef TTGO_TQ
   Rdelay = 0;
-  digitalWrite(LED, HIGH);
-  delayMicroseconds(50);
-  digitalWrite(LED, LOW);
+  //digitalWrite(LED, HIGH);
```
Comment (hpsaturn) [inline src/main.cpp]: @danielbernalb maybe are unnecessary comments too?

File: src/main.cpp [hunk 4]
```diff
@@ -655,5 +667,6 @@ void loop(){
   statusLoop();    // update sensor status GUI
   otaLoop();
   gui.pageEnd();
-  delay(400);
+  //delay(400);
```
Comment (hpsaturn) [inline src/main.cpp]: @danielbernalb I think that the sensor loop wait 1 second for each read, the main loop should be works well to 400ms

File: src/main.cpp [hunk 5]
```diff
@@ -267,9 +279,9 @@ void batteryloop() {
     return;
   }
   delayMicroseconds(1600);
-  digitalWrite(LED, HIGH);
-  delayMicroseconds(50);
-  digitalWrite(LED, LOW);
+  //digitalWrite(LED, HIGH);
```
Comment (hpsaturn) [inline src/main.cpp]: @danielbernalb maybe are unnecessary comments too?

File: src/main.cpp (hunk 1 above)
Comment (danielbernalb) [reply src/main.cpp]: Yes, is better in other site like the gui

File: src/main.cpp (hunk 2 above)
Comment (danielbernalb) [reply src/main.cpp]: Yes, is a problem founded after the last rev, the visualization of the serial frame.
In Visual code looks good but in Realterm and Arduino serial terminal the visualization is very confused.
The debug of the frame is very difficult and I choose to use the old routine for this test.

File: src/main.cpp (hunk 3 above)
Comment (danielbernalb) [reply src/main.cpp]: yes

File: src/main.cpp (hunk 5 above)
Comment (danielbernalb) [reply src/main.cpp]: yes

File: src/main.cpp (hunk 4 above)
Comment (danielbernalb) [reply src/main.cpp]: With 400ms the esp32 don´t work, even with 800ms generates a continuous reset at the beginning of the code and nothing works. I think is for the change in the serial routine to the original.

File: src/main.cpp (hunk 2 above)
Comment (hpsaturn) [reply src/main.cpp]: while we find a possible solution on the terminals on windows, I going to revert to original devel change

**Pull request merged**